tests/AttackTest.php

```diff
@@ -135,51 +128,42 @@ public function testInCheck(): void
 
     public function testInCheckmate(): void
     {
-        $chess = new ChessPublicator();
-        self::assertFalse($chess->inCheckmate());
-
-        $chess->load('r1bqk1nr/pppp1Qpp/2n5/2b1p3/2B1P3/8/PPPP1PPP/RNB1K1NR b KQkq - 0 4');
+        $chess = new ChessPublicator('r1bqk1nr/pppp1Qpp/2n5/2b1p3/2B1P3/8/PPPP1PPP/RNB1K1NR b KQkq - 0 4');
         self::assertTrue($chess->inCheckmate());
     }
 
     public function testInStalemate(): void
     {
-        $chess = new ChessPublicator();
-        self::assertFalse($chess->inStalemate());
-
         // fen source: https://www.redhotpawn.com/forum/only-chess/interesting-stalemate-position.152109
         // start fen : 3b3k/p6p/1p5P/3q4/8/n7/PP6/K4Q2 w - - 0 1
-        $chess->load('7k/p6p/1p3b1P/3q4/8/n7/PP6/K7 w - - 0 2');
+        $chess = new ChessPublicator('7k/p6p/1p3b1P/3q4/8/n7/PP6/K7 w - - 0 2');
         self::assertTrue($chess->inStalemate());
     }
 
     public function testInsufficientMaterial(): void
     {
-        $chess = new ChessPublicator();
-        self::assertFalse($chess->insufficientMaterial());
-
         // k vs k
-        $chess->clear();
+        $chess = new ChessPublicator('8/8/8/8/8/8/8/8 w - - 0 1');
         $chess->put(new Piece(Piece::KING, Piece::WHITE), 'e1');
         $chess->put(new Piece(Piece::KING, Piece::BLACK), 'e8');
         self::assertTrue($chess->insufficientMaterial());
 
         // k vs kn
-        $chess->clear();
+        $chess = new ChessPublicator('8/8/8/8/8/8/8/8 w - - 0 1');
```

Comment:
why not `Board::EMPTY`?

Reply:
or even better `new ChessPublicator()`

Reply:
> why not `Board::EMPTY`?

You're right. I started with a plain string, then added constant (and forgot to update something...)